etc/config/ForwarderCfg.yaml

```diff
@@ -8,29 +8,22 @@ LOGGING_DIR: /var/log/iip
 WORK_DIR: /var/tmp/data
 
 # daq partition name
-PARTITION: ats
+PARTITION: ncsa-comcam
+FOLDER: .templates
 
 # LCA-13501 segment order
-SEGMENT_ORDER: [ 10, 11, 12, 13, 14, 15, 16, 17, 07, 06, 05, 04, 03, 02, 01, 00 ]
-
-# @deprecated_soon
-# daq readout order for each sensor, currently WFS is the only one active. This
-# will be deprecated because readout order depends on Science, Wavefront, or
-# Guiding Sensor but not on vendor type such as ITL, E2V or STA.
-ITL: [ 00, 01, 02, 03, 04, 05, 06, 07, 10, 11, 12, 13, 14, 15, 16, 17 ]
-E2V: [ 10, 11, 12, 13, 14, 15, 16, 17, 07, 06, 05, 04, 03, 02, 01, 01 ]
-STA: [ 00, 01, 02, 03, 04, 05, 06, 07, 10, 11, 12, 13, 14, 15, 16, 17 ]
-WFS: [ 00, 01, 02, 03, 04, 05, 06, 07, 10, 11, 12, 13, 14, 15, 16, 17 ]
+DAQ_MAPPING: [ 10, 11, 12, 13, 14, 15, 16, 17, 07, 06, 05, 04, 03, 02, 01, 00 ]
+HDR_MAPPING: [ 10, 11, 12, 13, 14, 15, 16, 17, 07, 06, 05, 04, 03, 02, 01, 00 ]
```

Comment:
Are we going to need a separate ForwarderCfg.yaml file for ComCam that's different from LATISS because the mappings in the CCDs might not be the same? Further, is it possible that the mappings might be different from CCD to CCD in the ComCam?

Reply:
It is possible that different daq could give segment orders. Mappings cannot be different from ccd to ccd.